packages/ui/src/app/Providers.tsx

```diff
@@ -26,8 +27,10 @@ export const Providers = ({ children }: Props) => (
               <MembershipContextProvider>
                 <HashRouter>
                   <ModalContextProvider>
-                    <GlobalStyle />
-                    {children}
+                    <OnBoardingProvider>
+                      <GlobalStyle />
+                      {children}
+                    </OnBoardingProvider>
```

Comment:
I think that we have this provider on a very minimum set of components. Just to not pollute the globals too much.

I'd remove this provider from here. Also, I think that the onboarding modal _might_ not need to be global, a locally included modal should be OK.